package/nativeComponents/surfaces/NativeAppBar.js

```diff
@@ -8,8 +8,8 @@ export default function NativeAppBar(props) {
   const { auth } = props;
```

Comment:
Remove this

Reply:
Thanks for mentioning!

Reply:
Also remove `// eslint-disable-next-line no-unused-vars`